tests/acceptance/home/account/account-test.js

```diff
@@ -0,0 +1,66 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+module('Acceptance | user', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    this.sessionService = this.owner.lookup('service:session');
+    const store = await this.owner.lookup('service:store');
+
+    const loggedUser = await this.server.create('user');
+    const loggedUserModel = await store.findRecord('user', loggedUser.id);
+
+    this.set('store', store);
+    this.set('sessionService.currentUser', loggedUserModel);
+
+    window.localStorage.setItem(
+      'storage:logged-as',
+      JSON.stringify({ id: loggedUserModel.id })
+    );
+  });
+
+  test('visiting /account', async function (assert) {
+    await visit(`/account`);
+    assert.strictEqual(currentURL(), `/account`);
+
+    assert.dom('[data-test-input-name]').exists();
+    assert.dom('[data-test-input-surname]').exists();
+    assert.dom('[data-test-input-username]').exists();
+    assert.dom('[data-test-input-password]').exists();
+    assert.dom('[data-test-input-email]').exists();
+    assert.dom('[data-test-input-blik-number]').exists();
+    assert.dom('[data-test-input-bank-account-number]').exists();
+    assert.dom('[data-test-input-revolut]').exists();
+    assert.dom('[data-test-input-paypal]').exists();
+    assert.dom('[data-test-input-paypal]').exists();
+    assert.dom('[data-test-button-edit]').exists();
```

Comment:
Consider to remove this assertions which you check value further - there under the hood before this check `exists()` is invoke as well